crates/nu-cli/src/repl.rs

```diff
@@ -188,6 +189,18 @@ pub fn evaluate_repl(
         );
     }
 
+    if engine_state.get_config().show_startup_time && !engine_state.get_config().show_banner {
+        let dd = Color::Default.dimmed().prefix().to_string();
+        println!(
+            "{dd}startup time: {}",
```

Comment:
If we land this, it would be good to use title case like we do in the banner for consistency
```suggestion
            "{dd}Startup Time: {}",
```